configs/el8stream/el8stream-provision-engine.sh.in

```diff
@@ -6,7 +6,7 @@ rpm --import https://download.copr.fedorainfracloud.org/results/ovirt/ovirt-mast
 dnf --repofrompath=ovirt-master-snapshot,https://download.copr.fedorainfracloud.org/results/ovirt/ovirt-master-snapshot/centos-stream-8-x86_64/ \
     install -y dnf-utils ovirt-release-master
 
-dnf module enable -y javapackages-tools pki-deps postgresql:12
+dnf module enable -y javapackages-tools pki-deps postgresql:12 mod_auth_openidc:2.3
```

Comment:
don't we need this for rhel too?

Reply:
I don't think so because keycloak packages that require this module are not supposed to be 'required'/installed on rhel.